tests/test_diagnostics_database.py

```diff
@@ -48,9 +48,9 @@ def test_example_cdd(self):
 
         data = did.get_data_by_name('Diagnostic_Identification')
         self.assertEqual(data.name, 'Diagnostic_Identification')
-        self.assertEqual(data.start, 64)
+        self.assertEqual(data.start, 71)
```

Comment:
Note to other reviewers: For consistency with the DBC items, and compatability with the shared message codecs, using the same saw-tooth startbit numbering convention is necessary.